peekingduck/cli.py

```diff
@@ -291,16 +295,16 @@ def _create_nodes_from_config_file(
     config_path: str, project_dir: Path, node_type_choices: click.Choice
 ) -> None:
     """Creates custom nodes declared in the config file."""
-    run_config_path = Path(config_path)
-    if not run_config_path.is_absolute():
-        run_config_path = (project_dir / run_config_path).resolve()
-    if not run_config_path.exists():
+    pipeline_path = Path(config_path)
+    if not pipeline_path.is_absolute():
+        pipeline_path = (project_dir / pipeline_path).resolve()
+    if not pipeline_path.exists():
         raise FileNotFoundError(
-            f"Config file '{config_path}' is not found at {run_config_path}!"
+            f"Config file '{config_path}' is not found at {pipeline_path}!"
```

Comment:
Pipeline file instead to match the tutorials?

And code wise, can `config_path` not found at `pipeline_path` generate confusion to readers?

Reply:
Good point, let me take a look at this again.